internal/app/go-filecoin/plumbing/msg/previewer_test.go

```diff
@@ -1,60 +0,0 @@
-package msg
-
-import (
-	"context"
-	"testing"
-
-	bstore "github.com/ipfs/go-ipfs-blockstore"
-
-	"github.com/filecoin-project/go-filecoin/internal/pkg/consensus"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/repo"
-	th "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers"
-	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/actor"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/actor/builtin"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/address"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestPreview(t *testing.T) {
```

Comment:
It's not clear why this test has to go. It uses fake actors and everything. We probably will want to retain this functionality (the user's ability to preview a message execution). Could you attempt to restore it?